### PR TITLE
add logo, background image, layout css styling

### LINKED ISSUE #34: Add logo, background image, layout css styling
- [x] add logo
- [x] add background image
- [x] add navbar styling
- [x] add footer styling
- [x] change font

### PR DESCRIPTION
**Pull request by bpietraga**
Connected to #34, closes #34

### CODE REVIEW

File: app/views/layouts/_footer.html.slim [hunk 1]
```diff
@@ -1,7 +1,7 @@
 footer.sticky-footer
   .container
     .grid.grid-pair
-      .grid-item class="1/2"
-        p team III
-      .grid-item class="1/2"
-        p Internship
+      .grid-item class="1/2 mobile-1/1"
+        p Board&Roll is licensed under MIT licensce. See github repo.
```
Comment (trueskawka) [inline app/views/layouts/_footer.html.slim]: typo - should be 'license'

File: app/assets/stylesheets/_defaults.scss [hunk 2]
```diff
@@ -1,13 +1,47 @@
+@import 'https://fonts.googleapis.com/css?family=Slabo+27px';
 @import "hocus-pocus/scss/colors";
 
+$rwd-type: desktop-first;
+$rwd-map: (mobile: 480px, tablet: 720px, desktop: 1201px);
```
Comment (trueskawka) [inline app/assets/stylesheets/_defaults.scss]: a bit unusual breakpoints, why 1201px for desktop? just want to make sure it was tested

Comment (trueskawka) [inline app/assets/stylesheets/_defaults.scss]: explained

**Comment by trueskawka** [comment]
+1

File: app/assets/stylesheets/_defaults.scss (hunk 2 above)
Comment (bpietraga) [inline app/assets/stylesheets/_defaults.scss]: It is alright :) Suggested setup by hocus-pocus creator. All working

**Comment by gotar** [comment]
+1 ok just squash it

**Pull request merged**